WordPress/Classes/ViewRelated/Reader/ReaderSaveForLaterAction.swift

```diff
@@ -59,26 +57,14 @@ final class ReaderSaveForLaterAction {
         }
     }
 
-    private func presentPostSavedNotice(origin: ReaderSaveForLaterOrigin) {
-        let notice = Notice(title: Strings.postSaved,
-                            feedbackType: .success,
-                            actionTitle: Strings.viewAll,
```

Comment:
This code was showing an "View All" button for a "post saved for later" notice. If you are saving a post for later, why would you immediately want to see it? I checked Tracks, and, sure enough, 1-2 people were tapping it per day, likely by accident.